targets/xorg

```diff
@@ -226,7 +226,8 @@ if [ -n "$freon" -a -e "$inteldrv" ]; then
     if grep -F -q -a 'getuid0' "$inteldrv" 2>/dev/null; then
         # Hold xserver-xorg-video-intel to make sure that the modification
         # (which may have been made in a previous update) does not get erased.
-        apt-mark hold "$inteldriver"
+        # apt-mark hold "$inteldriver"
+        echo skipping apt-mark
```

Comment:
Instead of commenting out this line, please put and if statement around it:
```
if [ "${DISTROAKA:-"$DISTRO"}" = 'debian' ]; then
    apt-mark hold "$inteldriver"
fi
```